lib/vulcan.js

```diff
@@ -265,6 +266,7 @@ Vulcan.prototype = {
       }
       return this.loader.request(uri).then(function(content) {
         if (content) {
+          content = encodeString(content);
```

Comment:
I don't think so. If a user is inlining scripts, I think they are going to expect this behavior.